### PR TITLE
feat: Implement system packages attestation for Debian and RPM backends

### PR DESCRIPTION
**Pull request by fkautz**
Signed-off-by: Frederick F. Kautz IV <frederick@testifysec.com>

### CODE REVIEW

**Comment by fkautz** [comment]
Will add a subject then we will be good for review and merge.

File: attestation/system-packages/system-packages.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+		return "", "", "", err
+	}
+
+	return "linux", distribution, version, nil
+}
+
+// Attest implements attestation.Attestor.
+func (a *Attestor) Attest(ctx *attestation.AttestationContext) error {
+	os, dist, version, err := a.backend.DetermineOSInfo()
+	fmt.Println(os)
```
Comment (colek42) [inline attestation/system-packages/system-packages.go:124]: Can we use the logger here?  I have a console logger I am using to test this branch.  https://github.com/in-toto/go-witness/pull/408

**Review by colek42 — COMMENTED** [review]
This looks pretty good.  I'd like to see some table tests to test the parsing functions, check for edge cases, and use of the logger rather than fmt.

**Review by colek42 — APPROVED** [review]
